Rakefile

```diff
@@ -38,7 +38,17 @@ task :debug_compiler do
 end
 
 task :debug do
-  ruby "-I./lib -ropenssl -ve'puts OpenSSL::OPENSSL_VERSION, OpenSSL::OPENSSL_LIBRARY_VERSION'"
+  ruby_code = <<~'EOF'
+    libre_version_number_str = (!OpenSSL::LIBRESSL_VERSION_NUMBER.nil?) ?
+      OpenSSL::LIBRESSL_VERSION_NUMBER.to_s(16) : "nil"
+    puts <<~'MSG'
+    OpenSSL::OPENSSL_VERSION: #{OpenSSL::OPENSSL_VERSION}
+    OpenSSL::OPENSSL_LIBRARY_VERSION: #{OpenSSL::OPENSSL_LIBRARY_VERSION}
+    OpenSSL::OPENSSL_VERSION_NUMBER: #{OpenSSL::OPENSSL_VERSION_NUMBER.to_s(16)}
+    OpenSSL::LIBRESSL_VERSION_NUMBER: #{libre_version_number_str}
+    MSG
+  EOF
+  ruby %Q(-I./lib -ropenssl -ve'#{ruby_code}')
```

Comment:
I will update the indent and the style of the heredocs here. aligning with the current heredocs style in this repository and [the style recommended by Rubocop](https://github.com/rubocop/ruby-style-guide#heredocs).

Reply:
Rebased.

Reply:
Okay. I just wanted to align with other parts of the heredocs style in the ruby/openssl.

Reply:
If there is no policy for the heredocs, I would like 2 spaces indent for the heredocs because it's more readable for me. Let me modify it.

Reply:
Rebased with the 2 indent heredocs.